CONTRIBUTING.md

```diff
@@ -0,0 +1,41 @@
+# Coding guidelines
```

Comment:
The file should probably be named `CODING_GUIDELINES.md` rather than `CONTRIBUTING.md`. This file should probably point to that way and say that we expect you to follow it.